spec/models/user_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
- [ ] Please make sure that you write unit tests for your custom `set_post_counter_zero` since any custom methods in the models should have a corresponding unit test. :pray:

Reply:
This change has been implemented well